### ISSUE TITLE
Error after entering server address

### ISSUE THREAD

**Issue by mathBeard** [stefan-niedermann/nextcloud-notes#591]
Hello,
This is my first time submitting an issue on github, so please let me know if you need any additional information. I just recently set up a NextCloudPi server on my local network and after installing the latest version of Notes (0.27.1), I am not able to connect. 

After entering my server IP address and moving to the next field, I get a prompt to review the certificate which is the following error...

"Failed resolution of: Landroidx/databinding/DataBinderMapperImpl;..."

![Screenshot20190928_nc_notes](https://user-images.githubusercontent.com/55899453/65819828-d132b600-e1e6-11e9-8d7e-1214fdf41ed3.png)

The same error occurred on my Samsung Galaxy S7 and my wife's Motorola z4.

At first I was thinking that it had to do with my server, but now I'm wondering if there is a problem with the latest version of the Notes app since it works for me now after installing re-installing the previous version.

Let me know if you need any additional information.

Thank you.

**Comment by stefan-niedermann** [comment]
Thank you very much for reporting!
This might be related to the update of `cert4droid`-dependency by @korelstar (https://github.com/stefan-niedermann/nextcloud-notes/pull/582).

A few questions:

1. I assume you are using a self signed certificate?
2. Does sign in with an old version and then upgrading to `0.27.1` work or fail? 
3. Did you get the version from Play Store or F-Droid (should be identical, but still interesting)

Regards

**Comment by mathBeard** [comment]
1. Yes, I am using a self signed certificate.
2. Tried it, and yes! I was able to get both my phone and my wife's phone working with the new version by first installing the older version (0.25.0), signing into my server, accepting the X509 certificate, and then updating the app to the 0.27.1 version.
3. F-Droid

The only thing I wonder about now is whether or not I will get the error again if my server IP changes since it is not static. I'm assuming I will need to accept the certificate again at that point, and it looks like that is where the error comes in.
